vocab/credentials/v2/vocabulary.yml

```diff
@@ -163,7 +163,7 @@ property:
 
   - id: relatedResource
     label: Related resource
-    domain: cred:VerifiableCredential
+    domain: [cred:VerifiableCredential, cred:VerifiablePresentation]
```

Comment:
Given this points to credentials by URL, which can't you use a Data URI here?

```
{
  "@context": [
    "https://www.w3.org/ns/credentials/v2",
    "https://www.w3.org/ns/credentials/examples/v2"
  ],
  "id": "urn:uuid:5ec137ea-871e-11ee-a783-ef96a4397a9a",
  "type": ["VerifiablePresentation", "ExamplePresentation"],
  "relatedResource": [
    "data:application/jwt;base64,QzVjV...RMjUK==",
    "data:application/cwt;base64,ZmlOW...pYzMK="
 ]
}
```

Reply:
confusing when comparing to https://github.com/w3c/vc-data-model/pull/1372/files#diff-dfe00db9d510a45f71289312990ee36df870f36c02094551bfba625dcd0d8e96R193

Reply:
@decentralgabe, would you find this mechanism more acceptable? https://github.com/w3c/vc-data-model/pull/1379